person.rb

```diff
@@ -22,6 +23,11 @@ def correct_name
     @name
   end
 
+  def add_rental(rental)
+    @rentals.push(rental)
+    rental.person = self
+  end
```

Comment:
- [Optional] We can also have something like this:

```ruby
  def add_rental(book, date)
     # *The person itself should be sent as a parameter to create the new rental achieving the 'has-many' association
      Rental.new(date, book, self)
  end
```

Reply:
Will do, thank you